jekyll/_ccie/high-availability.md

```diff
@@ -250,31 +233,33 @@ settings.conf:
 }
 ```
 
-Note that you can dump the contents of settings.conf from an existing CircleCI installation by running `replicated app <app id> settings` on the services box. 
-
-license.rli is the CircleCI Enterprise license file. 
+It is also possible to retrieve the contents of `settings.conf` from an existing CircleCI installation by running `replicated app <app id> settings` on the Services machine. The `license.rli` is the CircleCI Enterprise license file. 
 
-## Troubleshooting
+### Troubleshooting
 
-### Known Issues
+* The various configuration files must be added to the ASG host before Replicated is installed. It may be possible to speed up the initial boot time by pre-pulling the CircleCI docker containers and adding them into the AMI. 
```

Comment:
What is the AWG host? If you mean the Services box I would say so, otherwise, I'm not sure what you mean. AutoScaling groups are run by AWS, they don't have a host.